README.md

```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

Comment:
This sounds a bit odd (is the intention to highligiht a "downgrade" of role from maintainer to approver?). 
Maybe it should be:
```suggestion
To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular contributor to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

Reply:
ah, lol... brain not good! Thanks